src/EnergyPlus/HeatPumpWaterToWaterCOOLING.hh

```diff
@@ -68,7 +69,7 @@ namespace HeatPumpWaterToWaterCOOLING {
     {
         // Members
         std::string Name; // user identifier
-        int WWHPPlantTypeOfNum;
+        DataPlant::PlantEquipmentType WWHPPlantTypeOfNum;
```

Comment:
looking at this DataPlant header, where should this new coil enum class be located to minimize/eliminate #includes? There will be a quiz.

// EnergyPlus Headers
#include <EnergyPlus/Data/BaseData.hh>
#include <EnergyPlus/DataGlobals.hh>
#include <EnergyPlus/DataLoopNode.hh>
#include <EnergyPlus/EnergyPlus.hh>
#include <EnergyPlus/OutputProcessor.hh>
#include <EnergyPlus/Plant/CallingOrder.hh>
#include <EnergyPlus/Plant/Enums.hh>
#include <EnergyPlus/Plant/Loop.hh>
#include <EnergyPlus/Plant/PlantAvailManager.hh>
#include <EnergyPlus/Plant/ReportLoopData.hh>

Reply:
😅 It should be outside the Plant/ directory since it's going to be used outside Plant as well if I understood right. I'm going to guess DataLoopNode.hh, since the others look like worse choices, but I'm not sure 😅  I went on a wild goose chase yesterday trying to fill up PlantEquimentTypes in Enums.hh with all component types possible, and eventually ended up in tears.

Reply:
I am guessing DataGlobals.hh but have not chased it out. You got the idea.

Reply:
I have not checked lately, is Enums,hh something we are moving towards? and if so adds another choice of location.

Reply:
There is also probably one big list of coils coming up very soon in [DataHVACGlobals.hh](https://github.com/NREL/EnergyPlus/blob/2ad67318cfe34c038d73081846e141c43440678b/src/EnergyPlus/DataHVACGlobals.hh#L147-L185). But this would mean adding DataHVACGlobals.hh to the list of includes in UnitarySystem. I am also not sure if it is an exhaustive list, even though it says all coils.